tensorboard/webapp/widgets/linked_time_fob/linked_time_fob_controller_test.ts

```diff
@@ -401,5 +408,52 @@ describe('linked_time_fob_controller', () => {
         end: {step: 3},
       });
     });
+
+    it('changing start input modifies start step', () => {
+      let fixture = createComponent({
+        linkedTime: {start: {step: 1}, end: null},
+      });
+      fixture.detectChanges();
+
+      let mainDiv = fixture.debugElement.query(By.css('linked-time-fob div'));
```

Comment:
You should consider giving the fob elements a class or id to better identify them. This will make it a little more certain you are getting the correct object. For example:

const startFob = fixture.debugElement.query(By.css('linked-time-fob.startFob'));

Reply:
I like that better. I always hesitate to add things to none test files for the purpose of testing but, in this case I think it is a good idea.